app/views/pages/gdprpolicy.ejs

```diff
@@ -0,0 +1,71 @@
+    <li>You have given Articleblog permission to do so</li>
+    <li>Processing your personal information is in Articleblog legitimate interests</li>
+    <li>Articleblog needs to comply with the law</li>
+</ul>
+  
+<p>Articleblog will retain your personal information only for as long as is necessary for the purposes set out in this Privacy Policy. We will retain and use your information to the extent necessary to comply with our legal obligations, resolve disputes, and enforce our policies.</p>
+
+<p>If you are a resident of the European Economic Area (EEA), you have certain data protection rights. If you wish to be informed what Personal Information we hold about you and if you want it to be removed from our systems, you can do so through your account settings at any time.</p>
+<p>In certain circumstances, you have the following data protection rights:</p>
+<ul>
+    <li>The right to access, update or to delete the information we have on you.</li>
+    <li>The right of rectification.</li> 
+    <li>The right to object.</li>
+    <li>The right of restriction.</li>
+    <li>The right to data portability</li>
+    <li>The right to withdraw consent</li>
+</ul>
+
+<h2>Log Files</h2>
+
+<p>Articleblog follows a standard procedure of using log files. These files log visitors when they visit websites. All hosting companies do this and a part of hosting services' analytics. The information collected by log files include internet protocol (IP) addresses, browser type, Internet Service Provider (ISP), date and time stamp, referring/exit pages, and possibly the number of clicks. These are not linked to any information that is personally identifiable. The purpose of the information is for analyzing trends, administering the site, tracking users' movement on the website, and gathering demographic information.</p>
+
+<h2>Cookies and Web Beacons</h2>
+
+<p>Like any other website, Articleblog uses 'cookies'. These cookies are used to store information including visitors' preferences, and the pages on the website that the visitor accessed or visited. The information is used to optimize the users' experience by customizing our web page content based on visitors' browser type and/or other information.</p>
+
+<p>For more general information on cookies, please read <a href="https://www.cookieconsent.com/what-are-cookies/">"What Are Cookies"</a>.</p>
+
+
+
+<h2>Privacy Policies</h2>
+
+<P>You may consult this list to find the Privacy Policy for each of the advertising partners of Articleblog.</p>
+
+<p>Third-party ad servers or ad networks uses technologies like cookies, JavaScript, or Web Beacons that are used in their respective advertisements and links that appear on Articleblog, which are sent directly to users' browser. They automatically receive your IP address when this occurs. These technologies are used to measure the effectiveness of their advertising campaigns and/or to personalize the advertising content that you see on websites that you visit.</p>
```

Comment:
DO we use third-party ad servers?

Reply:
Any library imported via CDN is classified as a third party server, the wording here relating to ad servers merely serves as an example to the reader to understand the techniques and why such techniques are required.